README.md

```diff
@@ -30,6 +30,20 @@ MATTR Wallet SDK React Native / [Exports](modules.md)
 
 # Getting started
 
+## How to get access to MATTR Pi Wallet SDK
+To gain access to the MATTR Pi Wallet SDK, please follow these steps. 
+
+1. Request or download the ["Terms of Agreement"](Licence.pdf).
+2. Read the "Terms of Agreement", sign it, and return it to us. 
+3. Create an account at [NPMJS - Node Package Manager for JavaScript](https://www.npmjs.com).
+4. Ensure multi-factor authentication (MFA) is Configured on NPMJS Account.
```

Comment:
@kimattr "Configured" should be "configured"

Reply:
@dylan-paul Done. Fixed in the latest commit.